c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
These are 2 different messages (surprisingly).
With JRE I still have the warnings you mention but proton builds. Without JRE the build fails for me.
Can you try it with raspbian?